ITEM.tsv

```diff
@@ -8523,261 +8523,261 @@ ITEM_20151102_008522	Unsent Letters
+ITEM_20151224_008600	Power Staff
+ITEM_20151224_008601	 - Staff attack (C Key) will increase the damage by 200% of the user's magic attack
+ITEM_20151224_008602	You can face foe without magic with this staff.
+ITEM_20151224_008603	Royal Pistol
+ITEM_20151224_008604	Superior Royal Pistol
+ITEM_20151224_008605	Superior Rapier
+ITEM_20151224_008606	Dual Rapier
+ITEM_20151224_008607	Superior Rapier
+ITEM_20151224_008608	Superior Cannon
+ITEM_20151224_008609	Practice Cannon
+ITEM_20151224_008610	Yorugis Cannon
+ITEM_20151224_008611	Superior Musket
+ITEM_20151224_008612	Practice Musket
+ITEM_20151224_008613	Yorugis Musket
+ITEM_20151224_008614	Royal Mage Glove
+ITEM_20151224_008615	Royal Mage Boots
+ITEM_20151224_008616	Royal Mage Pants
+ITEM_20151224_008617	Royal Mage Robe
+ITEM_20151224_008618	Bandit Glove
+ITEM_20151224_008619	Bandit Boots
+ITEM_20151224_008620	Bandit Pants
+ITEM_20151224_008621	Bandit Armor
+ITEM_20151224_008622	Royal Guard Glove
+ITEM_20151224_008623	Royal Guard Greaves
+ITEM_20151224_008624	Royal Guard Pants
+ITEM_20151224_008625	Royal Guard Armor
+ITEM_20151224_008626	Superior Royal Mage Gauntlet
+ITEM_20151224_008627	Superior Royal Mage Boots
+ITEM_20151224_008628	Superior Royal Mage Pants
+ITEM_20151224_008629	Superior Royal Mage Robe
+ITEM_20151224_008630	Superior Bandit Glove
+ITEM_20151224_008631	Superior Bandit Boots
+ITEM_20151224_008632	Superior Bandit Pants
+ITEM_20151224_008633	Superior Bandit Armor
+ITEM_20151224_008634	Superior Royal Guard Gauntlet
+ITEM_20151224_008635	Superior Royal Guard Greaves
+ITEM_20151224_008636	Superior Royal Guard Pants
+ITEM_20151224_008637	Superior Royal Guard Armor
+ITEM_20151224_008638	Belnia Monkey Leather Pants
+ITEM_20151224_008639	Belnia Monkey Leather Armor
+ITEM_20151224_008640	Elgos Glove
+ITEM_20151224_008641	Elgos Boots
+ITEM_20151224_008642	Elgos Pants
+ITEM_20151224_008643	Elgos Robe
+ITEM_20151224_008644	Evre Glove
+ITEM_20151224_008645	Evre Boots
+ITEM_20151224_008646	Evre Pants
+ITEM_20151224_008647	Evre Armor
+ITEM_20151224_008648	Grinas Gauntlet
+ITEM_20151224_008649	Grinas Greaves
+ITEM_20151224_008650	Grinas Leggings
+ITEM_20151224_008651	Grinas Armor
+ITEM_20151224_008652	Lets you to run fast as a wind.
+ITEM_20151224_008653	Many people does not know, but this equipment can be acquired so cheaply due to Goddess Laima planning with her old foresight. 
+ITEM_20151224_008654	Superior Sage Necklace
+ITEM_20151224_008655	Guardian Necklace
+ITEM_20151224_008656	Superior Guardian Necklace
+ITEM_20151224_008657	Nebelet Necklace
+ITEM_20151224_008658	Maven Necklace
+ITEM_20151224_008659	 - Hit by enemy's magic attack will activate effect increasing magic defense by {img red_up_arrow 16 16}6 and decreasing physical defense by {img green_down_arrow 16 16}6 (Maximum stack of 20)
+ITEM_20151224_008660	 - Increase magic amplification by 40% of HP recovery{nl} - Decrease basic HP recovery to 0
```

Comment:
- Increases magic amplification equal to 40% of HP recovery{nl}  - Decreases basic HP recovery to 0